Answers.md

```diff
@@ -0,0 +1,14 @@
+Question 1:
+    The class that has the most specificity weight is box3.
+Question 2:
+    The difference between display: block and display: inline is block takes up the whole container no matter if it fills up the container or not. Inline puts mulitiple things on one line till the line is filled up.
+Question 3:
+    The axsis is the cross axsis.
+Question 4:
+    Fixed website layout is not user friendly it doesn't change sizes as screen sizes change.
+    Fluid website layout is uses percentages to make the site be able to change with screen size change but can be challenging to make sure every screen size it being targeted.
+    Adaptive website layout is focuses on the user needs and adapts to different screen sizes easily, it relies on predefined screen sizes.
+    Responsive website layout is the most popular layout to use. It adapts to different screen sizes easily and is quick to load. 
+Question 5:
+    We use max-width because it sets the width for the whole layout to keep every piece responsive and user friendly to use.
```

Comment:
Great job! The key word that was used by Josh is constraint. Max-width provides a constraint and does exactly what you described it as.